examples/configurable-variables/configurable_variables.md

```diff
@@ -6,4 +6,4 @@ For more information, see [Configure a sample Ballerina service](/learn/configur
 
 ::: code configurable_variables.bal :::
 
-::: out configurable_variables.out :::
\ No newline at end of file
+::: out configurable_variables.out :::
```

Comment:
Nothing was changed here right? Or is this a bug?

Reply:
yeah I reverted the changes I did